src/index.ts

```diff
@@ -9,33 +9,38 @@ import {
 } from '@verdaccio/types';
 import {Issuer, Client, TokenSet} from 'openid-client';
 import asyncRetry = require('async-retry');
-import {Redis} from 'ioredis';
 import * as express from 'express';
 import {Express} from 'express';
 import {nanoid} from 'nanoid/async';
-import {createPool, Pool} from 'generic-pool';
 
-import {createRedis} from './redis';
 import {URL} from 'url';
 
+import { ISessionStorage, ITokenStorage } from './types';
+import { NewRedisStorage } from './redis';
+import { NewFileStorage } from './fs';
+
 type OidcPluginConfig = {
   publicUrl: string;
-  redisUri: string;
   issuer: string;
   clientId: string;
   clientSecret: string;
+  scope: string;
   usernameClaim?: string;
   rolesClaim?: string;
+
+  redisUri?: string;
+  fs_session_store_path?: string
+  fs_token_store_path?: string;
```

Comment:
These should be in camelCase for consistency

Reply:
Could we also have a single option instead of two?

Reply:
FIXED